mvc-api/search-core/src/main/java/org/sunbird/search/util/SearchConstants.java

```diff
@@ -112,4 +112,18 @@
     public static final String mode = "mode";
     public static final String softConstraints = "softConstraints";
     public static String soft = "soft";
+    public static String mlvectorurl = "http://127.0.0.1:1729/ml/vector/search";
+    public static String mlvectorListRequest = "{\"request\":{\"text\":[],\"language\":\"en\",\"method\":\"BERT\",\"params\":{\"dim\":768,\"seq_len\":25}}}";
+    public static String extraparams = "?fields=ageGroup,appIcon,artifactUrl,attributions,attributions,audience,author,badgeAssertions,board,body,channel,code,concepts,contentCredits,contentType,contributors,copyright,copyrightYear,createdBy,createdOn,creator,creators,description,displayScore,domain,editorState,flagReasons,flaggedBy,flags,framework,gradeLevel,identifier,itemSetPreviewUrl,keywords,language,languageCode,lastUpdatedOn,license,mediaType,medium,mimeType,name,originData,osId,owner,pkgVersion,publisher,questions,resourceType,scoreDisplayConfig,status,streamingUrl,subject,template,templateId,totalQuestions,totalScore,versionKey,visibility,year&orgdetails=orgName,email&licenseDetails=name,description,url";
+    public static String dikshaurl = "https://diksha.gov.in/";
+    public static String vidyadaanurl = "https://dock.sunbirded.org/";
+    public static String contentreadapi = "api/content/v1/read/";
+    public static String autocreatejobevent = "{\"eid\":\"BE_JOB_REQUEST\",\"mid\":\"\",\"actor\":{\"id\":\"Auto Creator\",\"type\":\"System\"},\"context\":{\"pdata\":{\"ver\":\"1.0\",\"id\":\"org.ekstep.platform\"},\"channel\":\"\"},\"object\":{\"ver\":\"1.0\",\"id\":\"\"},\"edata\":{\"action\":\"auto-create\",\"iteration\":1,\"objectType\":\"Content\",\"repository\":\"\",\"metadata\":{}}}";
+    public static String mvctopic = "sunbirddock.auto.creation.job.request";
+    public static String mvcFailedtopic = "sunbirddock.auto.creation.job.failedevent";
+    public static String contentKeysObj = "{\"board\":\"board\",\"medium\":\"medium\",\"grade\":\"gradeLevel\",\"subject\":\"subject\",\"chapterconceptname\":\"level1Concept\",\"topicconceptname\":\"level2Concept\",\"subtopicconceptname\":\"level3Concept\",\"chaptername\":\"level1Name\",\"topicname\":\"level2Name\",\"subtopicname\":\"level3Name\",\"textbookname\":\"textbook_name\",\"contenturl\":\"sourceURL\"}";
+    public static String csvcolumndelimeter = "|";
+    public static String csvfiledelimeter =",";
+    public static int csvRowsLimit = 1000;
```

Comment:
Please check and delete all the unused code. After a few days, no one understands what are these variables.